lib/themes/dosomething/paraneue_dosomething/js/components/onboard.js

```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+import Carousel from './carousel.js';
+
+const Onboard = React.createClass({
+  render: function() {
+    return (
+      <div className="onboard">
+        <Carousel />
+      </div>
+    );
+  }
+});
+
+if (localStorage.getItem('dosomething_enable_onboarding') == "true") {
```

Comment:
Using `window.localStorage` for low-fi tracking onboarding makes sense to me. 👍 You should be able to remove the client-side feature flag altogether if you conditionally include the bundle based on the Drupal feature flag.